appveyor.yml

```diff
@@ -47,6 +46,8 @@ environment:
     - PYTHON: "C:\\Python37-x64"
     - PYTHON: "C:\\Python38"
     - PYTHON: "C:\\Python38-x64"
+    - PYTHON: "C:\\Python39"
```

Comment:
may be time to drop PYTHON: "C:\\Python27" above?

Reply:
dropped!